BHoM_UI/Components/Engine/GetProperty.cs

```diff
@@ -66,8 +66,10 @@ public override object Run(object[] inputs)
         /**** Public Method               ****/
         /*************************************/
 
-        [Description("Get the value of a property with a given name from an object")]
-        [Output("Value of the property")]
+        [Description("Get the value of a property or of a list of properties from an object.")]
+        [Input("obj", "Object to extract the property value from. Can be a list of objects.")]
+        [Input("propName", "Name of the property to extract. Can be a list of names. \nTo know what property names you can specify, try connecting your object to a BHoM `Explode` component.")]
```

Comment:
Since we are specifying what the string can be, can we also explicitly say that you can sue the `.` operator?
e.g. for a `BH.oM.Structure.Bar`, you can use `StartNode.Position` to get the `BH.oM.Geometry.Point`